AUTHORS

```diff
@@ -0,0 +1,8 @@
+# This is the official list of Adversarial Robustness Toolbox (ART) Authors for copyright purposes.
+
+# Names should be added to this file as:
+# - Organization
+# - Person <email address>
+
+International Business Machines Corporation (IBM)
+Two Six Labs, LLC
```

Comment:
Should there be any year information connected to authors? (e.g., Two Six Labs, LLC - 2020, ...)

Reply:
Looking at other author files, adding year information doesn't seem to be too common, e.g. https://github.com/django/django/blob/master/AUTHORS